hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -152,18 +152,17 @@
       { name: "lc_usb",         gen: true,  type: "top", parent: "lc_src",  clk: "usb"     }
       { name: "sys",            gen: true,  type: "top", parent: "sys_src", clk: "main"    }
       { name: "sys_io",         gen: true,  type: "top", parent: "sys_src", clk: "io"      }
-      { name: "sys_io_div2",    gen: true,  type: "top", parent: "sys_src", clk: "io_div2" }
       { name: "sys_io_div4",    gen: true,  type: "top", parent: "sys_src", clk: "io_div4" }
-      { name: "sys_usb",        gen: true,  type: "top", parent: "sys_src", clk: "usb"     }
+      { name: "lc_usb",         gen: true,  type: "top", parent: "lc_src",  clk: "usb"     }
```

Comment:
huh... well this seems like a bug..apparently duplicated entries are not detected. You probably want to check for that and remove, or enhance the script to look for it.

Reply:
Thanks for double-check. Yes, I fixed this bug.